core/src/com/unciv/logic/map/Perlin.kt

```diff
@@ -54,6 +55,29 @@ object Perlin {
         return total/max
     }
 
+    fun ridgedNoise3d(x: Double, y: Double, z: Double,
```

Comment:
Please add a comment explaining what this is for, and what the parameters mean (or alternatively, link to an explanation)